dev/ParallaxView/APITests/ParallaxViewTests.cs

```diff
@@ -13,6 +13,7 @@
 using Windows.UI.Composition;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
```

Comment:
> Windows.UI.Xaml.Controls.Primitives [](start = 5, length = 36)

i thought scroller will not be in the OS side ?

Reply:
why one but not the other?

---
In reply to: [246476449](https://github.com/Microsoft/microsoft-ui-xaml/pull/61#discussion_r246476449) [](ancestors = 246476449)

Reply:
Keeping the ScrollViewer in WUCX was causing too many issues because of the existing one in the OS. It would have required a bunch of code-gen features for instance. 
The Scroller is still useful in the OS as it can be used as a standalone control or a building block for other controls.